### PR TITLE
Retriever CLI: Add dataset name to citation output

### PR DESCRIPTION
**Pull request by Deborah-Digges**
Currently, the output for the command `retriever citation all` is:

```
Citation:   Kissling WD, Dalby L, Flojgaard C, Lenoir J, Sandel B, Sandom C, Trojelsgaard K, Svenning J-C (2014) Establishing macroecological trait datasets:digitalization, extrapolation, and validation of diet preferences in terrestrial mammals worldwide. Ecology and Evolution, online in advance of print. doi:10.1002/ece3.1136
Description:   MammalDIET provides a comprehensive, unique and freely available dataset on diet preferences for all terrestrial mammals worldwide.

Citation:   Fritz, S. A., Bininda-Emonds, O. R. P. and Purvis, A. (2009), Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecology Letters, 12: 538-549. doi:10.1111/j.1461-0248.2009.01307.x
Description:   Mammal Super Tree from Fritz, S.A., O.R.P Bininda-Emonds, and A. Purvis. 2009. Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecology Letters 12:538-549
```

This change adds the dataset name to the output so that it makes the output in this particular case a little clearer:

```
Dataset:  MammalDIET
Citation:   Kissling WD, Dalby L, Flojgaard C, Lenoir J, Sandel B, Sandom C, Trojelsgaard K, Svenning J-C (2014) Establishing macroecological trait datasets:digitalization, extrapolation, and validation of diet preferences in terrestrial mammals worldwide. Ecology and Evolution, online in advance of print. doi:10.1002/ece3.1136
Description:   MammalDIET provides a comprehensive, unique and freely available dataset on diet preferences for all terrestrial mammals worldwide.

Dataset:  Mammal Super Tree
Citation:   Fritz, S. A., Bininda-Emonds, O. R. P. and Purvis, A. (2009), Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecology Letters, 12: 538-549. doi:10.1111/j.1461-0248.2009.01307.x
Description:   Mammal Super Tree from Fritz, S.A., O.R.P Bininda-Emonds, and A. Purvis. 2009. Geographical variation in predictors of mammalian extinction risk: big is bad, but only in the tropics. Ecology Letters 12:538-549
```

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ def main():
                 scripts = name_matches(script_list, args.dataset)
                 for dataset in scripts:
 
-                    print ("\nCitation:   {}".format(dataset.citation))
+                    print ("\nDataset:  {}".format(dataset.name))
```
Comment (henrykironde) [inline __main__.py]: Nice @Deborah-Digges , 
extra space

Comment (Deborah-Digges) [inline __main__.py]: @henrykironde Done! Removed the extra space

**Comment by henrykironde** [comment]
sorry for not being clear @Deborah-Digges. but could you please do something like this `print(` without the extra space :+1:

**Comment by Deborah-Digges** [comment]
done 👍

**Comment by henrykironde** [comment]
:+1: @Deborah-Digges

**Pull request merged**